models/files/lfda.R

```diff
@@ -0,0 +1,47 @@
+modelInfo <- list(label = "Local Fisher Discriminant Analysis",
+                  library = c("lfda"),
+                  type = "Classification",
+                  grid = function(x, y, len = NULL) {
+                    stop("grid is not available for lfda. ")
```

Comment:
This is always going to cause your model to fail. Caret models ALWAYS need a grid.  See here for a simple `modelInfo` you can copy: https://github.com/terrytangyuan/caret/blob/newlfda/models/files/lda.R